contracts/src/libraries/ownable.cairo

```diff
@@ -1,110 +1,155 @@
 use starknet::ContractAddress;
 
+// TODO: consider replacing with OwnableTwoStep if https://github.com/OpenZeppelin/cairo-contracts/pull/809/ lands
+
 #[starknet::interface]
-trait IOwnable<TContractState> {
-    fn owner(self: @TContractState) -> ContractAddress;
-    fn proposed_owner(self: @TContractState) -> ContractAddress;
-    fn transfer_ownership(ref self: TContractState, new_owner: ContractAddress);
-    fn accept_ownership(ref self: TContractState);
-    fn renounce_ownership(ref self: TContractState);
+trait IOwnable<TState> {
```

Comment:
just verifying it's not necessary to change the name of this generic variable?

Reply:
Yeah the name doesn't matter